src/ProductTable.jsx

```diff
@@ -0,0 +1,56 @@
+import "./ProductTable.css";
+
+function ProductTable({ x, y }) {
```

Comment:
X and Y make no sense! Let's name these variables something that describes their function.